listenbrainz/webserver/static/css/recommendation-page.less

```diff
@@ -0,0 +1,87 @@
+
+
+#recommendations {
```

Comment:
@MonkeyDo exactly my concerns

Reply:
@MonkeyDo Nice, I understand this now.  One concern I have is #listens has classNames like `listens-control` etc wherein `listens`  prefix/suffix is used. will it not be semantically wrong to use classNames that have `listens` in them for `recommendations` since `listens` and `recommendations` are two separate types. This was my thought process behind duplicating code.

Reply:
Good question.
I think we don't have to worry too much about that, considering one of two things will happen: 
1) we can rename the styles that will be used for both listens and recs (listen-controls -> controls, listen-card -> card or track-card, or something…)
2) some css will be generated but never used, which isn't too bad:
For example we'll get in the compiled CSS a rule like `#listens > .listen-card.playing-now, #recommendations > .listen-card.playing-now {…}` but we won't have any .playing-now element in the recs, and the css rule will just be ignored while the .listen-card.playing-now element in the #listens element will be styled as you expect.

Reply:
In short, what I suggest for now is to extend #listens, and we can clean up any unused css or refactor accordingly once things are a bit less in flux.